dali/test/python/test_video_reader_resize.py

```diff
@@ -160,6 +161,8 @@ def run_for_params(batch_size, video_reader_params, resize_params):
     compare_video_resize_pipelines(
         pipeline, gt_pipeline, batch_size, video_reader_params['sequence_length'])
 
+    gc.collect()
```

Comment:
Can you add a comment on what it is for here?

Reply:
Sure.